perun-base/src/main/java/cz/metacentrum/perun/core/api/CoreConfig.java

```diff
@@ -134,14 +134,6 @@ public void setReadOnlyPerun(boolean readOnlyPerun) {
 		this.readOnlyPerun = readOnlyPerun;
 	}
 
-	public boolean isCacheEnabled() {
-		return cacheEnabled;
```

Comment:
Couldn't we also remove the `cacheEnabled` field as well?

Reply:
You are right, fixed.